### DISCUSSION TITLE
Updating to a new version

### DISCUSSION THREAD

**Discussion by Kidcredo** [uxbox/uxbox#1261] [Q&A] [answered]
I am new to using Docker and recently started self hosting. 
I have a docker container running Penpot version 1.7 and would like to update to the latest version is there a guide that I can follow to do it safely without loosing my projects.

**Comment by niwinz** [comment, marked as answer]
There are no documentation right now, but the update is as simple as pulling a new image version. There are nothing special you need to do.

**Reply by Kidcredo** [reply]
Thanks. I was worried i would loose data

**Reply by niwinz** [reply]
This is the way we update ourselves our saas. So you should be safe, obviously you should perform a backup previous to update in any way (this is a general recommendation to any software update).

**Comment by jemedeiros** [comment]
My instance is stuck with v. 1.15.

I normally follow the instructions (https://help.penpot.app/technical-guide/getting-started/#update-penpot-1) and it always works.
I think the dockerhub image for 1.16 is not yet there?
